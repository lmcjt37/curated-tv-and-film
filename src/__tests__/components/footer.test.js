import React from 'react';
import { render, screen } from '@testing-library/react';

import Footer from '../../components/footer';

it('render all initial child components', async () => {
  render(<Footer />);

  expect(screen.getByTestId('footer')).toBeTruthy();
  expect(screen.getByTestId('footer-link-one')).toBeTruthy();
  expect(screen.getByTestId('footer-link-two')).toBeTruthy();
  expect(screen.getByTestId('footer-link-three')).toBeTruthy();
  expect(screen.getByTestId('footer-link-four')).toBeTruthy();
  expect(screen.getByTestId('footer-link-five')).toBeTruthy();
});

it('snapshot of initial component', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment).toMatchSnapshot();
});
