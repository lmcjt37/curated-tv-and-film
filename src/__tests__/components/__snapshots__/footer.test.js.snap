// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of initial component 1`] = `
<footer
  class="Footer-footer-1"
>
  <ul
    class="Footer-stackedList-3"
  >
    <li
      class="Footer-links-4"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        Made for Hacktoberfest ðŸŽƒ Â  Â© 2018-2021
      </p>
    </li>
    <li
      class="Footer-links-4"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        Owner Â 
        <a
          class="Footer-anchor-5"
          href="https://lmcjt.co.uk"
          rel="noopener noreferrer"
          target="_blank"
        >
          Luke Taylor
        </a>
      </p>
    </li>
  </ul>
  <ul
    class="Footer-inlineList-2"
  >
    <li
      class="Footer-links-4"
    >
      <a
        class="Footer-anchor-5"
        href="https://github.com/lmcjt37/curated-tv-and-film"
        rel="noopener noreferrer"
        target="_blank"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
        >
          Repository
        </p>
      </a>
    </li>
    <li
      class="Footer-links-4"
    >
      <a
        class="Footer-anchor-5"
        href="https://github.com/lmcjt37/curated-tv-and-film/graphs/contributors"
        rel="noopener noreferrer"
        target="_blank"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
        >
          Contributors
        </p>
      </a>
    </li>
    <li
      class="Footer-links-4"
    >
      <a
        class="Footer-anchor-5"
        rel="noopener noreferrer"
        target="_blank"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
        >
          build debug
        </p>
      </a>
    </li>
  </ul>
</footer>
`;
